src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -169,31 +171,46 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
       <ReactHookTextField
         required
         multiline
+        rows={4}
         control={control}
         label="Explain"
-        sx={{ flexGrow: 1, mx: 1, borderRadius: 2 }}
+        sx={{ flexGrow: 1, borderRadius: 2, width: '100%' }}
         {...register(`why.${index}.explain`)}
         errorMessage={errors.why?.[index]?.explain}
       />
     );
   };
 
   return (
-    <PageLayout title="New Change Request" previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]}>
-      <form
-        id={'create-standard-change-request-form'}
-        onSubmit={(e) => {
-          e.preventDefault();
-          e.stopPropagation();
-          handleSubmit(onSubmit)(e);
-        }}
-        onKeyPress={(e) => {
-          e.key === 'Enter' && e.preventDefault();
-        }}
+    <form
+      id={'create-standard-change-request-form'}
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        handleSubmit(onSubmit)(e);
+      }}
+      onKeyPress={(e) => {
+        e.key === 'Enter' && e.preventDefault();
+      }}
+    >
+      <PageLayout
+        stickyHeader
+        title="New Change Request"
+        previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]}
+        headerRight={
+          <Box textAlign="right" sx={{ mb: 2 }}>
+            <NERFailButton variant="contained" onClick={handleCancel} sx={{ mx: 1 }}>
+              Cancel
+            </NERFailButton>
+            <NERSuccessButton variant="contained" type="submit" sx={{ mx: 1 }}>
+              Submit
+            </NERSuccessButton>
+          </Box>
+        }
       >
-        <PageBlock title="Details">
-          <Grid container spacing={2}>
-            <Grid item xs={6}>
+        <Grid container spacing={2} display="flex" justifyContent="space-between">
+          <Grid container item spacing={2} xs={5} maxHeight={0} sx={{ mt: -5 }}>
```

Comment:
actually I know I told you to move it up, but I think you can remove the mt: -5

Reply:
we also want this to be responsive on smaller screens, so can you make the xs 12 on this one and the bottom one, and then you can make the current xs, mds. That way it'll look like this on bigger screens and stack the proposed solutions below on a smaller screen. Also remove the maxHeigh={0} it was causing some weird bugs locally